components/services/markdown/src/index.js

```diff
@@ -28,6 +28,14 @@ class ServiceMarkdown extends Component {
     )
   }
 
+  componentDidUpdate() {
+    const id = document.location.hash.substring(1)
+    if (id) {
+      const element = document.getElementById(id)
+      element && element.scrollIntoView({block: 'start', behavior: 'smooth'})
```

Comment:
FYI: smooth not supported by IE11

Reply:
I think he's aware about that. The good thing is that the same code works in IE11 but without behaviour.